questions.md

```diff
@@ -0,0 +1,5 @@
+# Remaining Questions About Pushes
+
+Hey, Chris.  Not really a question, but...
+
+I accidentally pushed a branch not named master first, and it messed with my head.  
```

Comment:
:joy: that definitely happens. It can help to have a nice utility like [Git Kraken](https://www.gitkraken.com) that will let you visually inspect your branches in those kinds of situations!